Source/Device.generated.swift

```diff
@@ -1191,3 +1190,47 @@ extension Device {
   }
 }
 #endif
+
+#if os(iOS)
+// MARK: - Apple Pencil
+extension Device {
+
+  /**
+    This option set describes the current Apple Pencils
+    - firstGeneration:  1st Generation Apple Pencil
+    - secondGeneration: 2nd Generation Apple Pencil
+   */
+  public struct ApplePencilSupport: OptionSet {
+
+    public var rawValue: UInt
+    
```

Comment:
Please remove the trailing whitespace on line 1206. I also suggest you set Xcode to automatically remove trailing whitespaces using this option:
https://stackoverflow.com/a/11830067/5544222

I would approve the review but I believe danger will just start to wine on other people if this isn't fixed.
```suggestion

```

Reply:
I pushed a new commit to resolve this as the issue stemmed from additional whitespace in the `Device.swift.gyb` file.